Cxp 657 new connect menu (#484)
akeneo/pim-api-docs: content/concepts/reference-entities.md

Note that the "Reference entity attribute" resource is not the same as the "Attribute" resource, even though they have a lot in common. The first one is used to describe reference entities records, the second one is use to describe products.
:::

> In the Akeneo UI, you can find the attributes that compose a given reference entity by following these simple steps. Go in the `Entities` menu. Click on a reference entity and then, click on `Attributes`. Below is an example of the attributes that compose the `Designer` entity in the UI.

Reviewer: ```suggestion
In the Akeneo UI, you can find the attributes that compose a given reference entity by following these simple steps. From the `Entities` menu, choose a reference entity and then, click on `Attributes`. Below is an example of the  `Designer` entity attributes in the UI.
```

Only attribute of type `single option` or `multiple options` can have options.

> In the Akeneo UI, if you want to manage the options of an reference entity attribute with single and multiple options:

Reviewer: ```suggestion
In the Akeneo UI, here are the steps to manage the options of a reference entity attribute with single and multiple options:
```

1. Select a reference entity in the `Entities` menu
1. Then, click on `Attributes`

> 1. Click on the edit button of a single or multiple options attribute. An edit popin should open. 

Reviewer: ```suggestion
1. Click on `Edit` for a single or multiple options attribute. An edit pop-in opens. 
```
